### PR TITLE
[composer] fix show_bcc and show_cc buttons

### PR DESCRIPTION
**Pull request by krisgardiner**
# Code changes

- [x] sets defaults for `show_cc` and `show_bcc` to `true`
- [x] fix functionality to toggle visibility for CC and BCC fields. 

# Screenshots

https://user-images.githubusercontent.com/34139730/145051692-00888609-bec9-4858-9a94-d2c527d662b8.mov

# Readiness checklist

- [x] Added changes to component `CHANGELOG.md`

# License

I confirm that this contribution is made under the terms of the MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by philrenaud — APPROVED** [review]

File: components/composer/src/Composer.svelte [hunk 1]
```diff
@@ -347,6 +347,9 @@
       triggerElement.removeAttribute("tabindex");
     }
   };
+
+  $: console.log({ _this });
+  $: console.log(!_this.show_cc, !!_this.show_cc_button, !!_this.show_to);
```
Comment (ozsivanov) [inline components/composer/src/Composer.svelte:352]: ✂️

File: components/composer/src/Composer.svelte [hunk 2]
```diff
@@ -652,19 +655,25 @@
             <button
               type="button"
               bind:this={ccField}
-              hidden={!_this.show_cc &&
+              hidden={_this.show_cc &&
```
Comment (ozsivanov) [inline components/composer/src/Composer.svelte:658]: Don't we want to hide this if `_this.show_cc` is false?

File: components/composer/src/Composer.svelte [hunk 3]
```diff
@@ -652,19 +655,25 @@
...
+                _this.show_cc = true;
+                previousProps = _this;
+              }}>CC</button
             >
 
             <button
               type="button"
               bind:this={bccField}
-              hidden={!_this.show_bcc &&
+              hidden={_this.show_bcc &&
```
Comment (ozsivanov) [inline components/composer/src/Composer.svelte:670]: Same thing here, this logic seems backwards

File: components/composer/src/Composer.svelte (hunk 2 above)
Comment (krisgardiner) [reply components/composer/src/Composer.svelte:658]: When `_this.show_cc` is `true` (cc field is visible) this button should be hidden. Please take a look at the recording to see it in action.

Comment (ozsivanov) [reply components/composer/src/Composer.svelte:658]: ohh sorry, this is the button 🤦 . My bad, I didn't expand the code to see what it was

**Review by ozsivanov — APPROVED** [review]

**Pull request merged**